### ISSUE TITLE
App crashes +logout problem

### ISSUE THREAD

**Issue by sanmiguel2019** [tinode/tindroid#94]
Android 8.1.0.

When I tap on photo icon to send an image, the app crashes.
When I tap on "Notifications" in the settings, the app crashes.

Logout works incorrectly. I am being logged out but after logout the app shows conversations list instead of login form. 

The problem persists in both ' devel and next branches.

**Comment by or-else** [comment]
This is not a useful bug report. Please fill out the form and provide logs.

**Comment by or-else** [comment]
There are no recent crashes in Crashlytics. That means you are running a modified version of the app and the crashes are likely due to the changes you made. Closing this issue. Please open another issue if you observe the crashes in a stock version of the app. Fill out the form if you decide to open another issue.

**Comment by sanmiguel2019** [comment]
The crash after tapping on photo icon  was caused by 

`<external-path name="tindroid_images" path="Android/data/co.tinode.tindroidx/files/Pictures" />`

line in `provider_paths.xml` file. As soon as I replaced `co.tinode.tindroidx` with my APP_ID, the problem disappeared.

Application still closes when I tap on "Notifications". Unfortunately, due to the hardware I can't run the app on my computer where I have Android studio installed, so, I can't provide logs.
